Dragster.cpp

```diff
@@ -1,91 +1,106 @@
+/*
+ * This file is a part of Dragster car set library.
+ *
+ * Realise: dragster object
```

Comment:
```suggestion
 *
```

Pretty obvious and repetitive things might be omitted